e2e/e2e_test.go

```diff
@@ -42,99 +40,39 @@ import (
 )
 
 var (
-	incrementFloat, _            = new(big.Int).SetString("1", 10)
 	amount               float64 = 400
 	hBarSendAmount               = hedera.HbarFrom(amount, "hbar")
 	tokensSendAmount             = 1000000000
 	hbarRemovalAmount            = hedera.HbarFrom(-amount, "hbar")
 	precision                    = new(big.Int).SetInt64(100000)
 	whbarReceiverAddress         = common.HexToAddress(receiverAddress)
-	memo                         = fmt.Sprintf("%s-0-0", receiverAddress)
+	memo                         = receiverAddress
 )
 
 const (
-	receiverAddress = "0x7cFae2deF15dF86CfdA9f2d25A361f1123F42eDD"
-	gasPriceGwei    = "100"
-	gasPrice        = "100000000000" // the Gas Price from above converted to WEI
-
+	receiverAddress         = "0x7cFae2deF15dF86CfdA9f2d25A361f1123F42eDD"
 	expectedValidatorsCount = 3
 )
 
 func Test_HBAR(t *testing.T) {
```

Comment:
I think that this test could do the `verifyTransferFromValidatorAPI` verification similar to how tokens are validating the response from the API